onnxruntime/core/providers/qnn/builder/qnn_def.h

```diff
@@ -48,6 +48,14 @@ enum class HtpPerformanceMode : uint8_t {
   kHtpBalanced,
 };
 
+// Defines the graph optimization strategy used by the HTP backend.
+enum class HtpGraphFinalizationOptimizationMode : uint8_t {
+  kDefault = 0,
+  kMode1 = 1,  // Faster preparation time, less optimal graph
```

Comment:
>kMode1 [](start = 2, length = 6)

How about level1, level2, level3

Reply:
I was thinking of using "level", but the word "level" implies that optimization increases as you up in level. However, that does not seem to be the case for these options. Mode 1 gives a less optimal graph in exchange for faster preparation time.

The specific wording the QNN headers use is "strategy".

Reply:
yea sounds similar to TensorRT where they call it algorithm tactics selection